portal/apps/org.wso2.carbon.iam.user-portal.app/src/main/modules/recovery-manager.js

```diff
@@ -264,4 +264,27 @@ var recoveryManager = {};
         return getRecoveryConfigs();
     };
 
+    recoveryManager.sendRecoveryNotification = function (uniqueUserId) {
+        if (uniqueUserId) {
+            try {
+                var recoveryMgtService = callOSGiService("org.wso2.is.portal.user.client.api.RecoveryMgtService", "setPasswordRecoveryNotification", [uniqueUserId]);
+            } catch (e) {
+                //todo need show error message in UI
+                Log.error(e + " ");
+            }
+        }
+    }
+
+    recoveryManager.updatePassword = function (code, password) {
+        try {
+            var passwordChar = Java.to(password.split(''), 'char[]');
+            var recoveryMgtService = callOSGiService("org.wso2.is.portal.user.client.api.RecoveryMgtService", "updatePassword", [code, passwordChar]);
+        } catch (e) {
+            //todo need show error message in UI
+            Log.error(e + " ");
```

Comment:
fixed https://github.com/wso2/product-is/pull/1127/commits/519f3eaee63647cf17a269037d8f7abaf0050c80